src/picongpu/include/particles/Particles.tpp

```diff
@@ -174,10 +174,18 @@ void Particles<T_ParticleDescription>::update(uint32_t )
         InterpolationScheme,
         fieldSolver::NumericalCellType > FrameSolver;
 
+    // adjust interpolation area in particle pusher to allow sub-sampling pushes
+    typedef typename PMacc::math::CT::add< LowerMargin,
+    typename PMacc::math::CT::make_Int<LowerMargin::dim, picongpu::UsedParticlePusher::extendMargin>::type
```

Comment:
Please use the trait `GetMargin<PUSHER>` instead to define a new variable in unitless. Each pusher can define it's own margin be specialize `traits::GetMagin`